cli/hooks.py

```diff
@@ -20,35 +22,106 @@ def chdir(dirname):
 
 class Hooks:
 
+    flag = False
+
     def __init__(self):
         self.utils = Utils()
+        self.whobj = WebHook()
 
     def run_hook(self, hookname, appdata, path, hook_input_metric):
         try:
             exit_code = 0
             function_execution_start_time = datetime.now()
+            defChannel = '#rogeros-deploy'
```

Comment:
Can we create a function like invoke_slack and move the slack related code into it. That way any code becomes more modularized. It is also easy to debug.